FileConv/Gfx/PrintShopFont.cs

```diff
@@ -0,0 +1,416 @@
+﻿/*
+ * Copyright 2023 faddenSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+using System.Diagnostics;
+
+using CommonUtil;
+using DiskArc;
+
+namespace FileConv.Gfx
+{
+    /// <summary>
+    /// Converts Print Shop clip art of various types.
+    /// </summary>
+    public class PrintShopFont : Converter
+    {
+        public const string TAG = "psfont";
+        public const string LABEL = "Print Shop Font";
+        public const string DESCRIPTION =
+            "Converts a Print Shop or Print Shop font file to a bitmap.  " +
```

Comment:
s/or Print Shop// (until the GS version works)

Reply:
Done.